raiden/tests/integration/test_provisioning.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import gevent
+
+from raiden.tests.utils.blockchain import wait_until_block
+
+
+@pytest.mark.parametrize('number_of_nodes', [6])
+@pytest.mark.parametrize('channels_per_node', [0])
+@pytest.mark.parametrize('cached_genesis', [False])
+@pytest.mark.parametrize('register_tokens', [False])
+def test_participant_selection(raiden_network, tokens_addresses):
+    token = tokens_addresses[0]
+
+    for num, app in enumerate(raiden_network):
+        manager = app.raiden.connection_manager_for_token(token)
+        chain = app.raiden.chain
+
+        wait_until_block(chain, chain.block_number() + 1)
+        manager.connect(100)
+        wait_until_block(chain, chain.block_number() + 1)
+
+    gevent.sleep(0)
+
+    connection_managers = [app.raiden.tokens_connectionmanagers[token] for app in raiden_network]
+    assert any(len(cm.open_channels) for cm in connection_managers)
+    assert all(len(cm.open_channels) for cm in connection_managers)
+
+    chain = raiden_network[-1].raiden.chain
+
+    def not_saturated(connection_managers):
+        return filter(
+            lambda cm: len(cm.open_channels) < cm.initial_channel_target,
```

Comment:
I know it's just an index in a lambda, but would it not be more readable to have `connection_manager` instead of `cm`?